libs/partners/google-vertexai/tests/integration_tests/test_chat_models.py

```diff
@@ -51,7 +51,7 @@ def test_candidates() -> None:
     assert len(response.generations[0]) == 2
 
 
-@pytest.mark.parametrize("model_name", ["chat-bison@001", "gemini-pro"])
+@pytest.mark.parametrize("model_name", ["chat-bison@001"])
```

Comment:
Given the differences between the available params for palm and gemini, would it make sense to continue to test this?

Reply:
I'd say "yes" for Palm.

Reply:
Why not worth testing gemini here?

Reply:
Sounds good. In the future, if the flakiness is something that should be fixed elsewhere, feel free to mark it as `@pytest.mark.xfail(...)`